cockatrice/src/dlg_settings.cpp

```diff
@@ -785,6 +803,7 @@ void DeckEditorSettingsPage::retranslateUi()
     urlLinkLabel.setText(QString("<a href='%1'>%2</a>").arg(WIKI_CUSTOM_PIC_URL).arg(tr("How to add a custom URL")));
     clearDownloadedPicsButton.setText(tr("Delete Downloaded Images"));
     resetDownloadURLs.setText(tr("Reset Download URLs"));
+    networkCacheLabel.setText(tr("Maximum image cache size:"));
```

Comment:
This label is too easily confused with the internal pixmap cache setting (on the general page see `pixmapCacheLabel`), in order to make the difference clearer I think it'd be best to move the two next to each other, either both on general or on the download sources page and have this one say it's on disk or otherwise make clear that one is persistent while the other is for the running application.

Reply:
Changed the label to mention "image directory" (should make it clear it's on disk), moved pixmap cache setting to "card source" page, and added tooltips to the QSpinBoxes to clarify further the purpose of both settings.